samples/grpc/local-drone-control-scala/src/main/resources/persistence.conf

```diff
@@ -0,0 +1,22 @@
+akka {
+  persistence {
+    journal {
+      plugin = "akka.persistence.r2dbc.journal"
+    }
+    snapshot-store {
+      plugin = "akka.persistence.r2dbc.snapshot"
+    }
+
+    r2dbc {
+      # FIXME mostly here to keep from lots of debug logs on start
```

Comment:
can that be filtered with logback instead, or should something be changed with the logging?

Reply:
On second thought I think we might not want all those topics for the small-local-drone-control service anyway, so maybe a low number here would be good. Just limiting resources.

Reply:
Just the one projection running, so changed it to 1